app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -11,7 +11,7 @@ related_content:
 promo_content:
 navigation: 20.50
 navigation_title: Get an international relocation payment
-navigation_description: Teachers of languages or physics could be eligible for financial help from the UK government worth £10,000.
+navigation_description: Teachers of languages or physics could be eligible for financial help from the UK government worth $nonuk_internationalrelocationpayment_value$.
```

Comment:
This should be showing on the category page, but is broken https://get-into-teaching-app-review-4027.test.teacherservices.cloud/non-uk-teachers. The value looks correct though?

Reply:
Nav descriptions need to use the other format as they appear on category pages which have a different file format (it's very confusing!)

There is an example on the internships PR https://github.com/DFE-Digital/get-into-teaching-app/pull/4028/files

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/d39a40c1-60ce-467d-b999-0db7b1323bba)

Reply:
Note to me change format of variable